### PR TITLE
Changes to source and output files in response to Phil's suggestions

### PR DESCRIPTION
**Pull request by hudson-ayers**

### CODE REVIEW

File: 6lo-Design-Considerations.txt [hunk 1]
```diff
@@ -13,15 +18,14 @@ Expires: December 29, 2018                                 June 27, 2018
...
-   (6LoWPAN) Internet Standard that would make it easier for
-   implementations to interoperate.
+   This document discusses guidelines for specifying low-power Internet
+   protocols to encourage implementation interoperability. These
+   guidelines are based around the importance of balancing memory usage
+   and energy efficiency, and the importance of not relying on Postel's
+   law when dealing with low resource devices. This document applies
+   these guidelines to the IPv6 over low-power wireless personal area
+   networks (6LoWPAN) Internet Standard, suggesting changes that would
+   make it easier for implementations to interoperate.
```
Comment (phil-levis) [inline 6lo-Design-Considerations.txt]: more likely for implementations to interoperate.

File: 6lo-Design-Considerations.txt [hunk 2]
```diff
@@ -13,15 +18,14 @@ Expires: December 29, 2018                                 June 27, 2018
...
-   for low power devices to encourage interoperability between diverse
-   implementations. These guidelines are based around the importance of
-   balancing memory usage and radio efficiency, and the importance of
-   not relying on Postel's law when dealing with low resource devices.
-   We evaluate and demonstrate these guidelines by using them to suggest
-   changes to the IPv6 over low-power wireless personal area networks
-   (6LoWPAN) Internet Standard that would make it easier for
-   implementations to interoperate.
+   This document discusses guidelines for specifying low-power Internet
+   protocols to encourage implementation interoperability. These
```
Comment (phil-levis) [inline 6lo-Design-Considerations.txt]: in order to improve implementation interoperability.

File: 6lo-Design-Considerations.txt [hunk 3]
```diff
@@ -226,29 +226,31 @@ Hudson Ayers           Expires December 29, 2018                [Page 4]
...
-   in a uniform manner. In fact, each specification implements a
-   different subset of the requirements in the 6LoWPAN specification,
-   and none implements the entire specification to the letter.  A
-   visualization of the mismatched feature support we discovered across
-   different 6LoWPAN implementations can be found in Table 1. Note that
-   OT is an abbreviation used throughout this document to refer to
-   OpenThread.
+   compression and fragmentation optimizations. Examining the code and
+   documentation for each of the aforementioned 6LoWPAN stacks reveals
+   that the stacks do not implement the specification in a uniform
```
Comment (phil-levis) [inline 6lo-Design-Considerations.txt]: uniformly

File: 6lo-Design-Considerations.txt [hunk 4]
```diff
@@ -366,15 +368,11 @@ INTERNET DRAFT         6lo Design Considerations           June 27, 2018
...
-   between stacks convincing, we now demonstrate that existing 6LoWPAN
-   stacks can actually generate the packet formats that we show other
-   stacks cannot receive. This proves that missing receive functionality
-   is not simply a case of limited 6LoWPAN stacks abstaining from
-   handling packets which no existing stacks ever generate anyway. 
+   These interoperability concerns are more than theoretical: existing
+   6LoWPAN stacks generate valid packets that other stacks discard. This
+   proves that missing receive functionality is not simply a case of
+   limited 6LoWPAN stacks abstaining from handling packets which no
+   existing stacks ever generate anyway.
```
Comment (phil-levis) [inline 6lo-Design-Considerations.txt]: ever generate.

File: 6lo-Design-Considerations.txt [hunk 5]
```diff
@@ -464,11 +462,11 @@ INTERNET DRAFT         6lo Design Considerations           June 27, 2018
    packet that might be sent by others.
 
    In practice, many 6LoWPAN implementations do not implement the entire
-   specification and, therefore, are not interoperable. We argue that
-   this is not a result of poor software design, but rather intentional
-   choices to implement different subsets of the specification that
-   favor limited RAM and code size, security concerns, and minimizing
-   engineering effort.
+   specification and, therefore, are not interoperable. One of the key
+   arguments in this document is that this is not a result of poor
```
Comment (phil-levis) [inline 6lo-Design-Considerations.txt]: Cut "One of the key arguments in this document is that"

**Review by phil-levis — COMMENTED** [review]
Overall, looks great. I'd suggest submitting it to the  WG today.

**Comment by hudson-ayers** [comment]
This version includes your changes, some more cleanup by me, and adds all references. I am going to see how it does with the idnits tool now

**Pull request closed without merging**